pteranodon/abstract_drone.py

```diff
@@ -591,6 +592,36 @@ def _join_thread(thread: Thread, timeout=1) -> None:
         except RuntimeError:
             pass
 
+    # method which waits for a given amount of time in seconds
+    def wait(self, wait_time: float, command=True) -> None:
+        """
+        A method which takes a float representing the amount of time to wait in seconds. This method will block
+        a thread until the time has elapsed.
+        :param wait_time: The amount of time to wait in seconds
+        :param command: Whether or not to add this to the command queue. If false this will block the main thread.
+        """
+        if command:
+            self._queue.appendleft((time.sleep, None, [wait_time], {}))
```

Comment:
Is the reason for the `appendleft` so that the waiting starts "immediately"? Could that mess up some other plugins that have added commands that may have a timeout? I think if I have a use case it might help more